### PR TITLE
added overview of tools

### PR DESCRIPTION
**Pull request by shaheen0b111**
includes a description of all docker,k8, kind, make components

### CODE REVIEW

File: _posts/2022-09-24-01-static-website-on-k8s-local.md [hunk 1]
```diff
@@ -14,7 +14,83 @@ Our tech industry has always promoted great innovations by adopting it for bette
...
+  - Docker Image can be called a static **template** or **blueprint** of an application that is created using instructions mentioned in a file called **Dockerfile**
+  - Docker Images are in a layered structure where each **RUN** command in **Dockerfile** will append new layer on top of previously executed command 
+- **Docker Container**<br>
+  - Running instance of Docker Image is called as **Container**
+  - Docker is also called as Containerization tool where application images are self-contained with all pre-requisites of source code, libraries, configurations, dependent packages, etc..
+- **Docker Registry**<br>
+  - [Docker Registry](https://docs.docker.com/get-started/overview/#docker-registries) is the storage for Docker Images from where we can pull and push docker images
+  - Docker Hub is the default public registry where the user's can push their local images or pull the publicly available remote images
+<br><br>
+![Docker Architecture](/assets/images/2022-09-22-00-docker-architecture.png)
```
Comment (alloftech) [inline _posts/2022-09-24-01-static-website-on-k8s-local.md]: Include source of image for giving credits.<br>
Check line 11 for reference

File: _posts/2022-09-24-01-static-website-on-k8s-local.md [hunk 2]
```diff
@@ -14,7 +14,83 @@ Our tech industry has always promoted great innovations by adopting it for bette
...
+- **kind**<br>
+  - **kind** called as **kubernetes in docker**
+  - There are multiple tools used which helps in creating k8 cluster on local system for setting up development environment
+    - [kind](https://github.com/kubernetes-sigs/kind)
+    - [minikube](https://github.com/kubernetes/minikube)
+    - [CodeReady Containers](https://github.com/code-ready/crc)
+    - [minishift](https://github.com/minishift/minishift)
+  - kind can create, manage , run the k8 local cluster using docker **container** as **k8 nodes**
+- **make**<br>
+  - make is a build tool which ease the creation of executables from source code or executes set of programs/instructions/commands which are written **Makefile**
```
Comment (alloftech) [inline _posts/2022-09-24-01-static-website-on-k8s-local.md]:
```suggestion
  - make is a build tool which ease the creation of executables from source code or executes set of programs/instructions/commands which are written in **Makefile**
```

File: _posts/2022-09-24-01-static-website-on-k8s-local.md [hunk 3]
```diff
@@ -14,7 +14,83 @@ Our tech industry has always promoted great innovations by adopting it for bette
...
 
-<!--more-->
+<br><br>
+
+## Overview of tools that will be in use for setting up a static website using kind
+
+- **Docker**<br>
+  - [Docker](https://docs.docker.com/get-started/overview/) is an open-source tool that provides feasibility to **create, package, run** the software/application independent of the underlying infrastructure
+  - It provides the best option to isolate the application and be executable irrespective of different environments
+  - Detailed [Docker architecture](https://docs.docker.com/get-started/overview/#docker-architecture)
```
Comment (alloftech) [inline _posts/2022-09-24-01-static-website-on-k8s-local.md]:
```suggestion
  - To understand docker thoroughly, please check upon this detailed [Docker architecture](https://docs.docker.com/get-started/overview/#docker-architecture) in their documentation.
```

File: _posts/2022-09-24-01-static-website-on-k8s-local.md [hunk 4]
```diff
@@ -14,7 +14,83 @@ Our tech industry has always promoted great innovations by adopting it for bette
...
+  - CLI commands for creating images
```
Comment (alloftech) [inline _posts/2022-09-24-01-static-website-on-k8s-local.md]: Need to use complete sentences for understanding

**Review by alloftech — CHANGES_REQUESTED** [review]
Please check upon commit suggestions & recommendations

File: _posts/2022-09-24-01-static-website-on-k8s-local.md (hunk 4 above)
Comment (shaheen0b111) [reply _posts/2022-09-24-01-static-website-on-k8s-local.md]: commit added

Comment (sankur-codes) [reply _posts/2022-09-24-01-static-website-on-k8s-local.md]: No changes seem to be done yet. Check if pushed.

File: _posts/2022-09-24-01-static-website-on-k8s-local.md (hunk 1 above)
Comment (shaheen0b111) [reply _posts/2022-09-24-01-static-website-on-k8s-local.md]: As required, links to images referred are added along with the source of it

File: _posts/2022-09-24-01-static-website-on-k8s-local.md (hunk 4 above)
Comment (shaheen0b111) [reply _posts/2022-09-24-01-static-website-on-k8s-local.md]: Added proper descriptive lines along with necessary links

File: _posts/2022-09-24-01-static-website-on-k8s-local.md (hunk 1 above)
Comment (shaheen0b111) [reply _posts/2022-09-24-01-static-website-on-k8s-local.md]: added caption for images

**Review by alloftech — APPROVED** [review]
After changes, LGTM

**Pull request merged**